pkg/plugin/driver/daemonset/daemonset.go

```diff
@@ -107,7 +107,7 @@ func (p *Plugin) createDaemonSetDefinition(hostname string, cert *tls.Certificat
 		Labels:      labels,
 		Annotations: annotations,
 		OwnerReferences: []metav1.OwnerReference{
-			metav1.OwnerReference{
```

Comment:
Just a gofmt change; I have my editor set to use `gofmt -s` which simplifies more than the default/minimum.